tests/pytype_blacklist.txt

```diff
@@ -1,13 +1,117 @@
+stdlib/3.4/asyncio/tasks.pyi
+stdlib/3/sqlite3/dbapi2.pyi
+stdlib/3.4/asyncio/__init__.pyi
+stdlib/3/sqlite3/__init__.pyi
+stdlib/3.4/asyncio/events.pyi
+stdlib/3/shelve.pyi
+
+# Non resolvable imports
+# imports non existing "builtins" module.
+stdlib/2/os/__init__.pyi
+# imports enum, which is undefined in Py2.7
+stdlib/2and3/plistlib.pyi
+
+# Unresolved dependencies
+# BadDependencyError: Can't find pyi for 'RegexFlag', referenced from 're'
+stdlib/3/re.pyi
+# BadDependencyError: Can't find pyi for '4.asyncio.coroutines', referenced
+# from '4.asyncio.locks'
+stdlib/3.4/asyncio/locks.pyi
+# BadDependencyError: Can't find pyi for '4.asyncio.futures', referenced from
+# '4.asyncio.queues'
+stdlib/3.4/asyncio/queues.pyi
+# BadDependencyError: Can't find pyi for '4.asyncio.events', referenced from
+# '4.asyncio.futures'
+stdlib/3.4/asyncio/futures.pyi
+# BadDependencyError: Can't find pyi for '4.asyncio.protocols', referenced
+# from '4.asyncio.streams'
+stdlib/3.4/asyncio/streams.pyi
+# BadDependencyError: Can't find pyi for 'PlistFormat', referenced from
+# 'plistlib'
+stdlib/2and3/plistlib.pyi
+# BadDependencyError: No Generator in module collections, referenced from
+# 'collections.abc'
+stdlib/3/collections/abc.pyi
+# SymbolLookupError: Couldn't find ConnectionResetError in http.client
+stdlib/3/http/client.pyi
+# SymbolLookupError: Couldn't find AST in ast
+stdlib/3/ast.pyi
+# BadDependencyError: Duplicate top level items:
+# 'concurrent.futures._base.Future, concurrent.futures._base.Future,
+# concurrent.futures._base.Future', referenced from
+# 'concurrent.futures.__init__'
+stdlib/3/concurrent/futures/__init__.pyi
+# BadDependencyError: Unreplaced NamedType: 'importlib.util'
+stdlib/3/importlib/__init__.pyi
+# BadDependencyError: No ChainMap in module typing, referenced from
+# 'collections.__init__'
+stdlib/3/collections/__init__.pyi
+# BadDependencyError: Can't find pyi for 'Sigmasks', referenced from 'signal'
+stdlib/3/signal.pyi
+# SymbolLookupError: Couldn't find ConnectionResetError in http.client
+stdlib/3/http/cookiejar.pyi
+stdlib/3/urllib/request.pyi
+# BadDependencyError: No _wrap_close in module os, referenced from 'pipes'
+stdlib/3/pipes.pyi
+# BadDependencyError: No stat_result in module os, referenced from 'posix'
+stdlib/3/posix.pyi
+
+
+# Conflicting values
+# ContainerError: Conflicting values for TypeVar
+# typing.ContextManager._T: multiprocessing.pool.Pool,
+# multiprocessing.pool.ThreadPool
+stdlib/3/multiprocessing/pool.pyi
+stdlib/3/multiprocessing/context.pyi
+stdlib/3/multiprocessing/process.pyi
+stdlib/3/multiprocessing/__init__.pyi
+stdlib/3/multiprocessing/synchronize.pyi
+
 # pytype doesn't yet support aliases with implicit type parameters
 # (e.g., here, FutureT = Future[T])
 stdlib/3.4/asyncio/tasks.pyi
+
+# ParseError: syntax error, unexpected ')', expecting NAME or NUMBER or
+# ELLIPSIS
+stdlib/2and3/locale.pyi
+stdlib/3/io.pyi
+stdlib/3/calendar.pyi
+stdlib/3/fcntl.pyi
+stdlib/3/gzip.pyi
```

Comment:
How come we didn't have to blacklist these before? The existing travis tests *do* check the pyi parsing step.

Reply:
That particular error is because of the `' '` default, right? That seems easy to fix at least.

Reply:
Created https://github.com/python/typeshed/pull/1721 to cahnge the defaults.

Reply:
We don't actually want to use typeshed's `builtins.pyi` with pytype, though, since we have our own (`__bulitin__.pytd`). We're probably just missing a symlink `builtins.pytd -> __builtin__.pytd` in `pytype/pytd/builtins`.

Reply:
It becomes a BadDependency Error once builtins.pyi is changed. As the pytype typing module does not have all the things needed.